README.md

```diff
@@ -102,6 +102,17 @@ This gRPC server is started by the driver running in _Node_ mode and implements
 The following diagram illustrates the communication channels between driver components:
 ![communication diagram](/docs/images/communication/pmem-csi-communication-diagram.png)
 
+### Security
+
+All communication is protected by mutual TLS. Both client and server must identify themselves and the certificate they present must be trusted. The common name in each certificate is used to identify the different components. The following common names have a special meaning:
+
+- `pmem-registry` is used by the [RegistryServer](#node-registry-server).
+- `pmem-node-controller` is used by [NodeControllerServers](#node-controller-server)
+
+The [`test/setup-ca-kubernetes.sh`](test/setup-ca-kubernetes.sh) script shows how to generate certificates signed by kubernetes cluster root Certificate Authority. And the provided [deployment files](deploy/kubernetes/pmem-csi.yaml) shows how to use the generated certificates to setup the driver. Test setup cluster already used this script to generate the certificates and used them to bring up the driver components. The [`test/setup-ca.sh`](test/setup-ca.sh) script also shows how to generate self signed certificates. These are just examples, administrators of a cluster must ensure that they choose key lengths and algorithms of sufficient strength for their purposes and manage certificate distribution.
+
```

Comment:
"The test cluster is set up using certificates created by that script." instead of "Test setup cluster already used ... to bring up the driver components."

If you  break up long paragraphs into multiple lines it becomes easier to comment on individual sentences and future diffs will be smaller.

Reply:
> "The test cluster is set up using certificates created by that script." instead of "Test setup cluster already used ... to bring up the driver components."
Done